docs/pages/guides/using-sentry.mdx

```diff
@@ -212,37 +213,64 @@ With `expo-updates`, release builds of both iOS and Android apps will create and
 
 - Your `release` will automatically be set to Sentry's expected value- `${bundleIdentifier}@${version}+${buildNumber}` (iOS) or `${androidPackage}@${version}+${versionCode}` (Android).
 - Your `dist` will automatically be set to Sentry's expected value: `${buildNumber}` (iOS) or `${versionCode}` (Android).
-- The configuration for build time sourcemaps comes from the `ios/sentry.properties` and `android/sentry.properties` files. For more information, refer to [Sentry's documentation](https://docs.sentry.io/clients/java/config/#configuration-via-properties-file). Manual configuration is only required for bare projects, the [sentry-expo config plugin handles it otherwise](#add-the-config-plugin).
-- Configuration for `expo publish` and `npx expo export` for projects is done via `app.json`, whether using bare workflow or not.
+- The configuration for build time sourcemaps comes from the **ios/sentry.properties** and **android/sentry.properties** files. For more information, refer to [Sentry's documentation](https://docs.sentry.io/clients/java/config/#configuration-via-properties-file). Manual configuration is only required for bare projects, the [sentry-expo config plugin handles it otherwise](#add-the-config-plugin).
+- Configuration for `expo publish` and `npx expo export` for projects is done via **app.json**, whether using bare workflow or not.
 
 Skipping or misconfiguring either of these can lead to invalid sourcemaps, and you won't see human-readable stacktraces in your errors.
 
-### Self-hosting updates
-
-If you're self-hosting your updates (this means you run `npx expo export` instead of `expo publish`), you need to:
-
-- replace `hooks.postPublish` in your **app.json** file with `hooks.postExport` (everything else stays the same)
-- add the `RewriteFrames` integration to your `Sentry.init` call like so:
+### Uploading sourcemaps for updates
+
+> This requires SDK 47, with `sentry-expo@6.0.0` or above and `expo-updates@0.15.5` or above.
+
+If you're using EAS Update, or if you're self-hosting your updates (this means you run `npx expo export` manually), you need to take the following steps to upload the sourcemaps for your update to Sentry:
+
+- Run `eas update`. This will generate a **dist** folder in your project root, which contains your JavaScript bundles and sourcemaps. This command will also output the 'Android update ID' and  'iOS update ID' that we'll need in the next step.
```

Comment:
```suggestion
- Run `eas update`. This will generate a **dist** folder in your project root, which contains your JavaScript bundles and sourcemaps. This command will also output the 'Android update ID' and 'iOS update ID' that we'll need in the next step.
```